### ISSUE TITLE
Cannot override message template when sending transactional emails

### ISSUE THREAD

**Issue by TrueWill** [customerio/customerio-node#74]
We are using `transactional_message_id` in `SendEmailRequest` to specify a message template, but we need to be able to override the `subject`.

Per the [Customer.io docs](https://customer.io/docs/transactional-api/#list-of-supported-parameters),

> If you include both a `transactional_message_id` and a `subject`, the `subject` in your request overwrites the values in your message template.

However this does not work with this library; this code is the culprit:

```js
    if ('from' in opts) {
      this.message.from = opts.from;
      this.message.subject = opts.subject;
      this.message.body = opts.body;
    }
```

Can you please set `subject` regardless?

This is a showstopper for us; we need to be able to set the email subject line to different values (but still use a template).

**Comment by TrueWill** [comment]
I did find a workaround:

If you set `from: undefined` in `SendEmailRequest`, the library will copy the `subject` to the message.

**Comment by TrueWill** [comment]
For others' sake, it might be worth reviewing the other properties that can be overwritten as well (please see the doc link above).

**Comment by mike-engel** [comment]
Fixed and published with `3.0.2`
